aspnetcore/migration/22-to-30.md

```diff
@@ -433,19 +433,38 @@ Libraries referencing [Microsoft.AspNetCore.Server.Kestrel.Https](https://www.nu
 
 Apps and libraries targeting ASP.NET Core 2.1 or later should remove any direct references to the [Microsoft.AspNetCore.Server.Kestrel.Https](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.Kestrel.Https) package.
 
-## Json.NET support
+<a id="jsonnet-support"></a>
 
-As part of the work to [improve the ASP.NET Core shared framework](https://blogs.msdn.microsoft.com/webdev/2018/10/29/a-first-look-at-changes-coming-in-asp-net-core-3-0/), [Json.NET](https://www.newtonsoft.com/json/help/html/Introduction.htm) has been removed from the ASP.NET Core shared framework.
+## Change default serializer back to Newtonsoft.Json (Json.NET)
```

Comment:
I slightly prefer the older title. It does more than change the default serializer, such as adding support for JsonPatch